packages/app-project/docker-compose.yml

```diff
@@ -2,31 +2,41 @@ version: '3.4'
 
 services:
   dev:
-    image: front-end-monorepo_dev:latest
+    image: front-end-monorepo_project-app:local
     build:
-      context: ../../
-      target: bootstrap
-    entrypoint:
-      - "yarn"
-      - "workspace"
-      - "@zooniverse/fe-project"
-    command: ["dev"]
+      context: .
+      args:
+        ASSET_PREFIX: https://fe-project.zooniverse.org
+        SENTRY_DSN: https://2a50683835694829b4bc3cccc9adcc1b@sentry.io/1492691
+    command: [yarn, "dev"]
+    environment:
+      - PORT=3000
+      - ASSET_PREFIX= http://localhost:3000
+      - SENTRY_DSN=https://2a50683835694829b4bc3cccc9adcc1b@sentry.io/1492691
```

Comment:
This doesn't have to be set for local dev builds if the SENTRY DSN is passed into the build via `args` - I left it here to show it can be modified at runtime, especially using `yarn dev`